tree/ntuple/test/ntuple_print.cxx

```diff
@@ -126,7 +126,7 @@ TEST(RNtuplePrint, ArrayAsRVec)
    testField.AcceptVisitor(visitor);
    std::string expected{std::string("") +
                         "$ Field 1       : arrayasrvecfield (ROOT::VecOps::RVec<float>)                 $\n" +
-                        "$   Field 1.1   : myfloat (float)                                              $\n"};
+                        "$   Field 1.1   : _0 (float)                                                   $\n"};
```

Comment:
This seems to be a degradation.  The user used on line 121 the name `myfloat` which is now no longer represented in the schema ... is that the intent?

Reply:
Humm .... when reading back how will the user properly retrieve `myfloat` vs `myotherfloat` if they are both turn into `_0` and the name are completely forgotten?
i.e. Is the name really gone (likely a blocker of sort) or is it just an artefact of the printing?